### PR TITLE
Add CSS and SVG processing to webpack

### PR DESCRIPTION
**Pull request by ESaracay**

### CODE REVIEW

**Review by Froggerson — APPROVED** [review]
looks good to me

File: src/main/java/com/google/sps/servlets/ReactServlet.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
   //If User does not exist in database, we create a new User entity. Page redirected to the /MyFolder Url
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException{
       request.setAttribute("TITLE", "Frame.cards");
-      ServletUtils.RenderReact("index", request, response);
+      ServletUtils.RenderReact("index2", request, response);
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/ReactServlet.java:17]: Choose a better name.

File: src/main/java/com/google/sps/servlets/ReactServlet2.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import com.google.sps.servlets.ServletUtils;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import javax.servlet.ServletException;
+
+@WebServlet(urlPatterns={"/myFolders","CreateCard","CreateFolder", "ImageInterface", "YoutubeInterface", "InsideFolder"})
+public class ReactServlet2 extends HttpServlet {
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/ReactServlet2.java:12]: Choose a better name for this servlet and this file.

File: frontend/src/App.js [hunk 3]
```diff
@@ -17,19 +17,22 @@ import PageError from "./pages/404";
...
-  return (
-    <div className="App">
+    return (
+      <div >
+        
       <Switch>
-        <Route exact path="/CreateCard" component={CreateCard} />
-        <Route exact path="/" component={MyFolders} />
+        <Route exact path="/CreateCard"><CreateCard></CreateCard></Route>
+        <Route exact path="/myFolders" exact component={MyFolders} />
```
Comment (tylerhou) [inline frontend/src/App.js:25]: Need consistent route naming.

File: frontend/src/index.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-//import css from  './components/App.css';
+import css from "./index.css";
 import App from './App';
 import { BrowserRouter } from "react-router-dom";
-
+/*
```
Comment (tylerhou) [inline frontend/src/index.js:6]: Delete code instead of commenting

File: frontend/src/index2.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
-//import './index.css';
-import Home from "./index2";
+import css from'./index.css';
```
Comment (tylerhou) [inline frontend/src/index2.js:3]: Does this file not compile if we import without a name?

File: frontend/src/index2.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
```
Comment (tylerhou) [inline frontend/src/index2.js:1]: This file should have a better name than `index2`.

File: frontend/src/main-components/Header.js [hunk 7]
```diff
@@ -41,21 +41,20 @@ const Header = (props) => {
    * If the device size is less than 700px wide then we will display
    * a button for our side menu instead of our logo.
    */
-  const width = window.innerWidth;
+   const width = window.innerWidth;
```
Comment (tylerhou) [inline frontend/src/main-components/Header.js:44]: Formatting.

File: frontend/src/main-components/Header.js [hunk 8]
```diff
@@ -41,21 +41,20 @@ const Header = (props) => {
...
   if (width < 700) {
```
Comment (tylerhou) [inline frontend/src/main-components/Header.js:45]: This should be done with CSS. Currently, if the width changes then this won't update.

https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

File: frontend/src/main-components/Sidebar.js [hunk 9]
```diff
@@ -34,14 +34,14 @@ const Sidebar = (props) => {
...
   return (
     <Side id="side" class="flex-container">
-      {/*
+      
       <SideIcon link="/" icon={homeIcon}></SideIcon>
       <SideIcon link="/CreateCard" icon={newCardIcon}></SideIcon>
       <SideIcon link="/CreateFolder" icon={newFolderIcon}></SideIcon>
       <SideIcon link="/ImageInterface" icon={objectIcon}></SideIcon>
       <SideIcon link="/YoutubeInterface" icon={youTubeIcon}></SideIcon>
       <SideIcon link="/noneyet" icon={accountIcon}></SideIcon>
```
Comment (tylerhou) [inline frontend/src/main-components/Sidebar.js:43]: `noneyet` shouldn't be in the main branch.

File: src/test/java/com/google/sps/LoginServletTest.java [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+public final class LoginServletTest {
+  
+  private final LocalServiceTestHelper helper =
+        new LocalServiceTestHelper(new LocalUserServiceTestConfig())
+          .setEnvIsAdmin(true).setEnvIsLoggedIn(true)
+          .setEnvEmail("test@gmail.com").setEnvAuthDomain("gmail.com");
+
+
+  // Need to update this servlet test
+  private static final User USER_A = new User("null", "/_ah/logout?continue\u003d%2F", "test@gmail.com", true);
```
Comment (tylerhou) [inline src/test/java/com/google/sps/LoginServletTest.java:42]: LOGGED_IN_USER_A?

File: src/test/java/com/google/sps/LoginServletTest.java [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+  private static final User USER_B = new User("/_ah/login?continue\u003d%2F", "null", "null", false);
```
Comment (tylerhou) [inline src/test/java/com/google/sps/LoginServletTest.java:43]: LOGGED_OUT_USER?

**Pull request merged**